.gitmodules

```diff
@@ -1,24 +1,24 @@
 [submodule "third-party/vcpkg"]
-	path = compiler+runtime/third-party/vcpkg
-	url = https://github.com/jank-lang/vcpkg.git
+        path = compiler+runtime/third-party/vcpkg
+        url = https://github.com/jank-lang/vcpkg.git
 [submodule "compiler+runtime/third-party/folly"]
```

Comment:
Very odd to see changes in here.

Reply:
Tabs to spaces. My editor converts them automatically on save.